sample-demo-lambda-app/lambda_function.py

```diff
@@ -54,3 +54,39 @@ def check_prime(num):
             return False
         sq_root += 1
     return True
+
+
+def sync_ddb_table(source_ddb, destination_ddb):
+    response = source_ddb.scan(
+        TableName="table1"
+        )
+    for item in response['Items']:
+        destination_ddb.put_item(
+            TableName="table2", Item=item
+            )
+
+
+def list_sns(region, creds, sns_topics=[]):
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Default values in Python are created exactly once, when the function is defined. If that object is changed, like the `sns_topics` object in this example, subsequent calls to the function will refer to the changed object, leading to confusion. We recommend that you set the default value to `None`, then check inside the function if the parameter is `None` before creating the desired mutable object.

[Learn more](https://docs.python.org/3/faq/programming.html#why-are-default-values-shared-between-objects)